apps/web-giddh/src/app/contact/contact.component.html

```diff
@@ -545,7 +537,7 @@ <h1 class="mr-b1">{{localeData?.type_message_body}}:</h1>
                                         <label></label>
                                         <input type="checkbox" (click)="selectAccount($event, account)"
                                             [(ngModel)]="account.isSelected"
-                                            [ngClass]="{'checkbox-visible': (selectedCheckedContacts.length || isMobileScreen), 'position-absolute': !selectedCheckedContacts.length}" />
```

Comment:
Class name getting replaced. Please check

Reply:
They do the same thing `position-absolute` is of Bootstrap and `pos-abs` is our custom. Will create a new PR that will replace all the class names with Bootstrap.